depthai_demo.py

```diff
@@ -871,5 +876,18 @@ def guiOnToggleDisparity(self, value):
                 if self.selectedPreview not in updated:
                     self.selectedPreview = updated[0]
                 self.updateArg("show", updated)
+    App().start()
+
+
+def runOpenCv():
+    confManager = prepareConfManager(args)
+    demo = Demo()
+    demo.run_all(confManager)
 
-    App().start()
\ No newline at end of file
+
+if __name__ == "__main__":
+    is_pi = platform.machine().startswith("arm")
```

Comment:
This might match Mac M1 (`"arm64"`), although that could be a good thing (not sure on the PySide6 availability). But anyway, for now in almost all cases we run with Rosetta, so it returns `"x86_64"`.

Reply:
@VanDavv 
I'd also try importing the pyside6 and catching on import if not found and automatically downgrading to `opencv` option

Also, might be worth checking doing the same for software backend (not sure if possible, as those looked like runtime errors from an available backend)

Reply:
@VanDavv So this also doesn't catch `aarch64`, and maybe shouldn't catch `arm64` (e.g Mac M1). Should we properly fix, or remove it, now as PySide6 import is checked?

Reply:
@alex-luxonis I changed the logic now, not relying on platform but whether PySide6 is importable or not